src/indexer/index.ts

```diff
@@ -1,13 +1,147 @@
-import indexerClient from "../config/indexer.config";
+import { CHAIN_NAME } from "@/config/algosigner.config";
+import {
+	GLOBAL_STATE,
+	GLOBAL_STATE_MAP_KEY,
+	LOCAL_STATE,
+	LOCAL_STATE_MAP_KEY,
+} from "@/constants/constant";
+import DaoID from "@/store/DaoID";
+import WalletStore from "@/store/WalletStore";
+import {
+	Key,
+	StateValue,
+	AccountAddress,
+} from "@algo-builder/algob/build/types";
+declare var AlgoSigner: any; // eslint-disable-line
```

Comment:
We defined algosigner type at [here](https://github.com/scale-it/algo-builder/blob/master/packages/web/src/algo-signer-types.ts). Could we use it?